src/services/config.service.ts

```diff
@@ -65,6 +71,7 @@ const configSchema: SchemaOf<Config> = object({
   features: featuresSchema.notRequired(),
   integrations: object({
```

Comment:
We are probably going to want to rethink how the 'integrations' config settings are set from the dashboard to prevent wonky configurations like having a full cleeng and full inplayer config set.  For now, we should probably at least throw an error in `validateConfig` if both are present and/or have `clientId` set.